### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by mishakoshilka**
- Deploy and create a Pull Request with a [DEMO LINK](https://mishakoshilka.github.io/layout_landing-page/)

### CODE REVIEW

**Comment by mishakoshilka** [comment]
[DEMO LINK](https://mishakoshilka.github.io/layout_landing-page/)

**Review by loralevitska — CHANGES_REQUESTED** [review]
A little more and you will be on target)

- add phone number by phone icon hover
image
- make sure that all interactive elements have a cursor pointer on hover
- remove default autocomplete background color, it breaks design 
image
- fix form submit it doesn't work now

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great work! Consider fixing validation errors: 
Screenshot 2023-08-03 at 16 50 02
Screenshot 2023-08-03 at 16 50 17
Screenshot 2023-08-03 at 16 50 23
Screenshot 2023-08-03 at 16 49 29
The menu in landscape mode should be scrollable
Screenshot 2023-08-03 at 16 48 28
There is a visible background, consider fixing it

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
1. Make all these fields are required
image

2. When I click send I get this error
image

3. Add transition for hover effects for these elements
image

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
1. Still not fixed from the previous review
image

2. When I click send page should not scroll on top page/
image

3. Also, data from the form does not need to be shown in the URL address
image

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
The background of these fields is different, also when I click on send there is no information that the field is mandatory
image

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,328 @@
...
+            <a href="#" class="top-bar__logo-link">
+              <img
+                src="images/logo.png"
+                alt="Logo Nothing"
+                class="top-bar__logo"
+              >
+            </a>
+            <div class="top-bar__icons">
+              <div class="container-for-phone">
+                <a href="tel:+1-234-5555-55-55" class="icon icon--phone"></a>
```
Comment (volodymyr-soltys97) [inline src/index.html:31]: Need to fix format phone everywhere only in href
```suggestion
                <a href="tel:+123455555555" class="icon icon--phone"></a>
```

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,328 @@
...
+    </header>
+
+    <aside class="page__menu menu" id="menu">
+      <div class="container">
+        <div class="top-bar menu__top">
+          <a href="#" class="top-bar__logo-link">
+            <img
+            src="images/logo.png"
+            alt="Logo Nothing"
+            class="top-bar__logo">
```
Comment (volodymyr-soltys97) [inline src/index.html:53]: Need to check and fix it everywhere
```suggestion
            <img
              src="images/logo.png"
              alt="Logo Nothing"
              class="top-bar__logo"
            >
```

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Almost done!
Let's improve code style

**Review by volodymyr-soltys97 — APPROVED** [review]
Well done 🔥

**Pull request closed without merging**